src/routes/+page.svelte

```diff
@@ -47,7 +47,9 @@
       Der digitale Fristenkalender bringt den bewährten
       Hochfrequenz-Fristenkalender vom Wandformat direkt in Ihre Kalender-App.
       So können alle offiziellen Fristen, beispielsweise der GPKE und MaBiS, in
-      Ihre Kalender importiert und dort als Ereignisse angezeigt werden.
+      Ihre Kalender importiert und dort als Ereignisse angezeigt werden. Wir
+      geben unser Bestes, dass alle Daten korrekt und vollständig sind, können
+      es aber leider nicht garantieren.
```

Comment:
Würde da ein Absatz reinmachen damit es einfacher zu sehen ist.

```suggestion
      Ihre Kalender importiert und dort als Ereignisse angezeigt werden.
      
      Wir geben unser Bestes, dass alle Daten korrekt und vollständig sind, können
      es aber leider nicht garantieren.
```